project-codeflare/codeflare-sdk
# Updated Ray version to 2.20.0

# Issue link


[RHOAIENG-6450](https://issues.redhat.com/browse/RHOAIENG-6450)
# What changes have been made

Updated CFSDK ray dependency to 2.20.0
# Verification steps



## Setup
### Notebook server ODH/RHOAI/Local
* Clone this repository with `git clone https://github.com/project-codeflare/codeflare-sdk.git`
* Checkout this PR's branch
* Run `poetry build` - install if needed (`pip install poetry`)
* Run `pip install --force-reinstall dist/codeflare_sdk-0.0.0.dev0-py3-none-any.whl` 
* Restart your notebook kernel
## Testing
Use this image for the following test scenarios: `quay.io/project-codeflare/ray:2.20.0-py39-cu118`
* Run through demo notebooks
* Ensure GPU utilization is working correctly
* Ensure basic & local interactive demos work correctly
* Ensure job submission works correctly

## Checks
- Testing Strategy
